spec/system/partners/family_requests_system_spec.rb

```diff
@@ -29,7 +29,7 @@
       click_link "Your Previous Requests"
       expect(page).to have_text("Request History")
       expect(Partners::ChildItemRequest.pluck(:child_id)).to match_array(children.pluck(:id))
-      expect(Partners::ItemRequest.pluck(:item_id)).to match_array(children.pluck(:item_needed_diaperid).uniq)
+      expect(Partners::ItemRequest.pluck(:item_id)).to match_array(children.map(&:needed_item_ids).flatten.uniq)
```

Comment:
None of the specs actually show that we now support multiple items... let's make sure we have some examples of that in all the specs that are relevant.

Reply:
addressed 2114d33536245cb529e118a8988c09cb914af835